picard/debug_opts.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2024 Laurent Monin
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+from collections import namedtuple
+from enum import IntEnum
+
+
+class DebugOpt(IntEnum):
+    WS_REPLIES = 1
+
+
+DebugOptDesc = namedtuple('DebugOptDesc', ('name', 'description'))
+
+debug_opts_descriptions = {
+    DebugOpt.WS_REPLIES: DebugOptDesc(name='ws_replies', description=N_('Log web service replies'))
+}
```

Comment:
Perhaps this could be modified slightly to include 'title' which would appear in the drop down list rather than the name that is used on the command line?  This title could also be translated in the drop down.

```suggestion
DebugOptDesc = namedtuple('DebugOptDesc', ('name', 'title', 'description'))

debug_opts_descriptions = {
    DebugOpt.WS_REPLIES: DebugOptDesc(name='ws_replies', title=N_('WS replies'), description=N_('Log web service replies'))
}
```

Then Line 174 in `logview.py` might be something like:

```
action = QtGui.QAction(_(desc.title), self, checkable=True, checked=debug_opt in debug_opts)
```

Reply:
Good idea, done in [6d15126](https://github.com/metabrainz/picard/pull/2391/commits/6d151260bd133e25b1998b304e2e0f10ca20b96f)